docs/docs/deploy/install-server.md

```diff
@@ -48,17 +48,18 @@ If the Automatic Install fails, try these steps to install Firezone manually.
 1. Edit the default configuration located at `/etc/firezone/firezone.rb`.
    We've chosen sensible defaults that should be a good starting point for most
    installations. For production installations, you'll want to specify your
-   FQDN and SSL certificate paths:
+   FQDN and enable ACME for certificate issuance and renewal:
```

Comment:
Sorry could you replace `FQDN` with `a valid external URL` -- this is an artifact leftover from a previous config setting